src/mon/MonCommands.h

```diff
@@ -301,14 +301,14 @@ COMMAND("mds versions",
 	"check running versions of MDSs",
 	"mds", "r", "cli,rest")
 COMMAND_WITH_FLAG("mds tell " \
-	"name=who,type=CephString " \
+	"name=role,type=CephString " \
```

Comment:
Let's use `role_or_gid` as this ends up in help output.

Reply:
Dropped `mds tell`.